lib/components/LazyVideo/LazyVideo.js

```diff
@@ -151,6 +153,13 @@ function _defineProperty(obj, key, value) {
   return obj;
 }
 
+if (typeof window !== 'undefined' && typeof document !== 'undefined') {
+} // var LazyLoad = require('vanilla-lazyload');
+// document.addEventListener("DOMContentLoaded", function() {
+//   yall();
```

Comment:
I'm not sure why the lib/ files didn't have the comments removed, but they're removed in the src files.